### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by eduardosancho**
# What was done in this PR: 🇨🇷 

- [x] Delete all boiler plate from CRA 🌷 
- [x] Create src/Components/Calculator.js 🔢 
- [x] Develop the UI of the calculator 🥨 
- [x] Use class based components 😎 
- [x] Renders in the browser without problems 🌞 

monkee

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './Calculator.css';
+
+export default function Calculator() {
+  return (
+    <>
```
Comment (hiromataba) [inline src/components/Calculator.js:6]: - [ ] Your `Calculator.js` component should be class-based, Please update that 

image

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```
Comment (hiromataba) [inline src/App.js:5]: - [ ] Your `App.js` component should be class-based, Please update that 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave: :wave: @eduardosancho  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :medal_military: 
- Nice PR title and Description :heavy_check_mark: 
- Awesome design :+1: 
- Professional README file :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @eduardosancho,

NICE GIF BTW!!!

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**